nx_parallel/algorithms/cluster.py

```diff
@@ -47,15 +47,18 @@ def _compute_clustering_chunk(node_iter_chunk):
     else:
         node_iter = list(G.nbunch_iter(nodes))
 
-    total_cores = nxp.cpu_count()
+    configs = nxp.get_curr_configs()
+    cores = nxp.cpu_count()
+    total_cores = cores if cores is not None else -1
+    configs["n_jobs"] = total_cores
 
     if get_chunks == "chunks":
         num_in_chunk = max(len(node_iter) // total_cores, 1)
         node_iter_chunks = nxp.chunks(node_iter, num_in_chunk)
     else:
         node_iter_chunks = get_chunks(node_iter)
 
-    result = Parallel(n_jobs=total_cores)(
+    result = Parallel(**configs)(
```

Comment:
not sure if `Parallel` should be used here or `parallel_config`...

or something like this(like in PR#https://github.com/networkx/nx-parallel/pull/7)
```.py
with joblib.parallel_backend('<parallel_backend_name>'):
    return joblib.Parallel(n_jobs=-1)(calls)
```

Reply:
`parallel_backend` is a deprecated functionalities; used `parallel_config` here.